app/views/pages/sandbox.html.slim

```diff
@@ -0,0 +1,46 @@
+- content_for :page_title_prefix do
+  | Sign in to Teaching Vacancies
+
+- content_for :breadcrumbs do
+  = govuk_back_link text: t("buttons.back"), href: root_path
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    markdown:
```

Comment:
I would suggest not to use markdown here, if you have a look at this page, for me it's much more readable without it

https://github.com/DFE-Digital/teaching-vacancies/blob/main/app/views/pages/job-application-preview.html.slim

but I'll leave this up to you. If you think this is more readable, I'll approve it like this